epoch/api/contract_api_usage.md

```diff
@@ -49,6 +49,8 @@ In order to affect the state of the chain you have to submit the signed transact
 ### Contract Create
 [/tx/contract/create doc](https://aeternity.github.io/epoch-api-docs/?config=https://raw.githubusercontent.com/aeternity/epoch/master/apps/aehttp/priv/swagger.json#/external/PostContractCreate)
 
+[/tx/contract/create/compute doc](https://aeternity.github.io/epoch-api-docs/?config=https://raw.githubusercontent.com/aeternity/epoch/master/apps/aehttp/priv/swagger.json#/external/PostContractCreateCompute)
```

Comment:
Testing by [replacing embedded URL of Swagger schema with commit](https://aeternity.github.io/epoch-api-docs/?config=https://raw.githubusercontent.com/aeternity/epoch/f2b09c9d10686bbee6adb0795a9749c1cdfb5b08/apps/aehttp/priv/swagger.json#/external/PostContractCreateCompute)) in [PR that is proposing Swagger schema](https://github.com/aeternity/epoch/pull/1418/commits), I see it works.